doc/radosgw/notifications.rst

```diff
@@ -75,41 +75,43 @@ added when the notification is committed to persistent storage.
 Topic Management via CLI
 ------------------------
 
-Configuration of all topics, associated with a tenant, could be fetched using the following command:
+Fetch the configuration of all topics associated with tenants by running the
+following command:
```

Comment:
Can more than one tenant be specified comma-separated, or with multiple instances of `--tenant=`?

Reply:
This is question for @yuvalif. I'll keep an eye on this PR, and when he answers this question, I'll return to this part of notifications.rst to ingest his answer.

Reply:
@zdover23 @anthonyeleven tenant has to be a single value. passing multiple `--tenant` parameters is not supported (I think the last one would override)